storage/tianmu/core/aggregation_algorithm.cpp

```diff
@@ -289,9 +287,17 @@ void AggregationAlgorithm::MultiDimensionalGroupByScan(GroupByWrapper &gbw, int6
           t->GetAttrP(i)->term.vc->LockSourcePacks(m);
         }
       }
+
+      [[maybe_unused]] const char *thread_type = "multi";
+
+#ifdef DEBUG_AGGREGA_COST
```

Comment:
a session level system be a good idea to turn on or turn off the trace. or you can use a command or statement to trace this feature. just like MySQL does.

Reply:
When this module is implemented in the future, the scheme will be redesigned according to the new scheme, and the current code will be kept temporarily